eth/watchers/orchestratorwatcher.go

```diff
@@ -97,9 +97,15 @@ func (ow *OrchestratorWatcher) handleTranscoderActivated(log types.Log) error {
 	}
 
 	if !log.Removed {
+		uri, err := ow.lpEth.GetServiceURI(transcoderActivated.Transcoder)
+		if err != nil {
```

Comment:
Let's add a test for this error case

Reply:
fixed in 285268c7, but also added a test for `GetTranscoder` error (in case log is removed) and opted to use a single `Err` field on the `StubEthClient` rather than creating a seperate field for each case (b3d0d034). 

Can rework some other tests to use this single `Err` field in a seperate PR but this is not a priority.